test/functional/wallet_migration.py

```diff
@@ -1196,6 +1196,60 @@ def test_disallowed_p2wsh(self):
 
         wallet.unloadwallet()
 
+    def test_miniscript(self):
+        # It turns out that due to how signing logic works, legacy wallets that have valid miniscript witnessScripts
+        # and the private keys for them can still sign and spend them, even though output scripts involving them
+        # as a witnessScript would not be detected as ISMINE_SPENDABLE.
+        self.log.info("Test migration of a legacy wallet containing miniscript")
```

Comment:
e41f47597d86022d5a9947ee0b08151d4336cffc: "containing scripts matching miniscript"?

Reply:
Miniscript is a subset of script, so this sentence is correct.

Reply:
> Miniscript is a language for writing (a subset of) Bitcoin Scripts 

https://bitcoin.sipa.be/miniscript/

So when I read "containing miniscript" I interpret that as containing a descriptor using the miniscript language. But these are legacy wallets, which don't have descriptors, so they can only contain script itself.